open_discussions/urls.py

```diff
@@ -51,21 +50,21 @@
     re_path(r"^api/token/refresh/", refresh_jwt_token),
     # React App
     re_path(r"^$", index, name="open_discussions-index"),
-    re_path(r"^auth_required/$", index),
-    re_path(r"^content_policy/$", index),
+    re_path(r"^infinite/", index),
+    re_path(r"^saml/metadata/", saml_metadata, name="saml-metadata"),
     re_path(
         r"^c/(?P<channel_name>[A-Za-z0-9_]+)/(?P<post_id>[A-Za-z0-9_]+)/"
         r"(?P<post_slug>{post_slug_pattern})/comment/(?P<comment_id>[A-Za-z0-9_]+)/?$".format(
             post_slug_pattern=POST_SLUG_PATTERN
         ),
-        channel_post,
+        index,
         name="channel-post-comment",
```

Comment:
Removing this `re_path` (and similar ones) causes a bunch of channel tests to break. Best removed as part of #2